integration/src/test/java/org/mule/test/integration/exceptions/ErrorHandlingWithFlowRefTestCase.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.integration.exceptions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import org.junit.Test;
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.processor.Processor;
+import org.mule.test.AbstractIntegrationTestCase;
+
+public class ErrorHandlingWithFlowRefTestCase extends AbstractIntegrationTestCase {
+
+  private static int executions;
+
+  @Override
+  public void doSetUp() throws Exception {
```

Comment:
use a different method annotated with `@Before` instead of overriding the before method from the superclass.